recipes/mesa/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "mesa" %}
+{% set version = "13.0.3" %}
+{% set sha256 = "d9aa8be5c176d00d0cd503cb2f64a5a403ea471ec819c022581414860d7ba40e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: mesa-{{ version }}.tar.xz
+  url: https://mesa.freedesktop.org/archive/{{ version }}/mesa-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - llvm-3.9.patch  # [win]
+
+build:
+  number: 0
+  skip: True  # [win and py<35]
+#   skip: True  # [win and not py27]
+#   msvc_compiler: "10.0"  # [win]
+#   features:   # [win and py>=35]
+#     - vc14    # [win and py>=35]
+  features:   # [win]
+    - vc14    # [win]
+
+requirements:
+  build:
+    - llvmdev
+    - mako  # [win]
+    - m2-flex  # [win]
+    - m2-bison  # [win]
+    - m2-xz  # [win32]
+    - m2w64-xz  # [win64]
+#     - vc 14  # [win and py>=35]
+    - vc 14  # [win]
+    - scons  # [win]
+
+# test:
```

Comment:
Not sure yet what to test. Maybe a simple GLX C program with Xvfb? Or be more picky and use piglit?